ONSdigital/dp-frontend-router
# Feature/handle errors

### What

Intercept error responses (>= 500) and render a standard error page.

It attempts to use the renderer (not currently done), and if there's an error, returns a styled 'disaster' page

Contains code from https://github.com/ONSdigital/dp-frontend-router/pull/11 - that should be reviewed first

### How to review

* Run dp-frontend-router without any other services running
* Open http://localhost:20000, should get a styled internal server error page
* Start other services (babbage, zebedee)
* Open http://localhost:20000, should get the babbage response page
* Stop zebedee to force a 500 response
* Open http://localhost:20000, should get the new styled error page rather than babbage error page

### Who can review

Anyone except @ian-kent
